x/tokenomics/keeper/settle_session_accounting.go

```diff
@@ -184,10 +183,11 @@ func (k Keeper) SettleSessionAccounting(
 	return nil
 }
 
-func (k Keeper) getCoinFromComputeUnits(ctx context.Context, root smt.MerkleRoot) sdk.Coin {
+func (k Keeper) getCoinFromComputeUnits(ctx context.Context, root smt.MerkleRoot, computeUnitsPerRelay uint64) sdk.Coin {
```

Comment:
NIT: thoughts on renaming this to "computeUnitsToCoins"?

Reply:
That would be confusing. It is the compute_units_per_relay param from the service, so imo the name is appropriate.

Reply:
1. But this function is on the receiver, not on the service.
2. There is no service in context (not the receiver, not the module, etc...)
3. This function is designed in a "functional" manner: provide computeUnits and get coins. We're not "getting it from the service"

Reply:
@Olshansk I can rename it if you'd like, but I don't get why the compute units per relay would be named compute units to coins. It isn't compute units for coins, it's compute units per relay. The compute units to tokens multiplier tokenomics param defines the coin for compute units.

Reply:
How about `computeUnitsPerRelayToNumCoins` and we add a `godoc` with a further explanation?

Reply:
@Olshansk ohhh, you are talking about the function name. The parameter was highlighted, so I thought your comment was about the `computeUnitsPerRelay` parameter. I couldn't figure out why you wanted to name it something completely separate from what it was. I'll change the function name as you suggest.

Reply:
> The parameter was highlighted

That's just hot GitHub works: highlighting the addition.

> I'll change the function name as you suggest.

👍